examples/Madgwick6DOF/Madgwick6DOF.ino

```diff
@@ -15,60 +15,58 @@ Gyroscope gyro(GYRO_ADDRESS_V1);
 // если напаяны перемычки, устройства доступны по новым адресам
 // Accelerometer accel(ACCEL_ADDRESS_V2);
 // Gyroscope gyro(GYRO_ADDRESS_V2);
- 
+
 // переменные для данных с гироскопов, акселерометров
 float gx, gy, gz, ax, ay, az;
- 
+
 // получаемые углы ориентации
 float yaw, pitch, roll;
```

Comment:
For the future, here the interface is no longer necessary to break it painfully "incompatible".
 
struct  Coordinates {float x; float y; float z};
struct  Angles {float yaw; float pitch; float roll};

Coordinates gyro, accel;

gyro = gyroscope.readRadPerSec(); // not gyroscop.readRadPerSecXYZ(x,y,z);


This is clearer